scripts/experience-decisioning/index.js

```diff
@@ -549,16 +608,24 @@ function adjustedRumSamplingRate(customOptions) {
 
 export async function loadEager(customOptions = {}) {
   sampleRUM.cases ||= {};
+  sampleRUM.cases.audiences = adjustedRumSamplingRate(customOptions);
```

Comment:
Essentially, by default only 1 out of 100 pages is sampled, with this we can adjust the sampling in the context of experiments/campaigns/etc. so that we gather data more quickly if those are meant to be "short-lived". This is something CCX is using and that we were asked to integrate

Reply:
Thanks for the explanation. Got it!...... but :)
in order for this code to work you need this [or condition](https://github.com/ramboz/franklin-experience-decisioning/blob/61475244b72e6489b07216789bd7b4e27954551f/scripts/lib-franklin.js#L54) in the boilerplate, otherwise the code won't be executed. 

We introduced recently in the boilerplate an similar mechanism: https://github.com/adobe/helix-project-boilerplate/blob/a774acc651359137422ea734d014164509ae5940/scripts/lib-franklin.js#L78

If you update to latest boilerplate, you could reuse it, something like:
```
sampleRUM.always.on('experiment', (data) => {
  if (!window.hlx.rum.isSelected) {
     // track experiments with higher sampling rate
    window.hlx.rum.weight = Math.min(window.hlx.rum.weight, pluginOptions.rumSamplingRate);
    window.hlx.rum.isSelected = (window.hlx.rum.random * window.hlx.rum.weight < 1);
    if (window.hlx.rum.isSelected) {
       sampleRUM('experiment',data);
    }
  }
});
``` 

And last question, what is this line `sampleRUM.drain('stash', sampleRUM);` doing?

Reply:
To be fair… I'm not sure myself. Added this as per Lar's request and based on https://github.com/adobe/express-website/blob/787405b88d50d07b5fdb04ab36cefeb5ba9ba80f/express/scripts/scripts.js#L84-L90

But I like your code better 😉 so will adjust accordingly